concepts/05 Widgets/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,185 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/). 
+
+To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RequiredRule/) to it.
+
+An editor is validated when its value changes. To validate all editors simultaneously, call the [validate()](/Documentation/ApiReference/UI_Widgets/dxForm/Methods/#validate) Form method. 
+
+In the following example, we specify the [isRequired](Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#isRequired) property for the `Name` item. We also specify the **NumericRule** for `officeNumber` and **EmailRule** for `Email` items.
```

Comment:
Will be here a screenshot? Or, a link to WG to review how validation is show?

Reply:
I don't think that it is required here since the most important part here is the internal validation engine but not the appearance. We also give a link to WG in the last section.